sequencer/dbmanager.go

```diff
@@ -178,18 +195,50 @@ func (d *dbManager) GetWIPBatch(ctx context.Context) (*WipBatch, error) {
 	}
 
 	wipBatch := &WipBatch{
-		batchNumber:  lastBatch.BatchNumber,
-		coinbase:     lastBatch.Coinbase,
-		accInputHash: lastBatch.AccInputHash,
-		// initialStateRoot: lastBatch.StateRoot,
+		batchNumber:    lastBatch.BatchNumber,
+		coinbase:       lastBatch.Coinbase,
+		accInputHash:   lastBatch.AccInputHash,
 		stateRoot:      lastBatch.StateRoot,
+		localExitRoot:  lastBatch.LocalExitRoot,
 		timestamp:      uint64(lastBatch.Timestamp.Unix()),
 		globalExitRoot: lastBatch.GlobalExitRoot,
+	}
+
+	txs, _, err := state.DecodeTxs(lastBatch.BatchL2Data)
+	if err != nil {
+		return nil, err
+	}
+
+	txTrackerArray := make([]TxTracker, len(txs))
 
-		// TODO: txs
-		// TODO: remainingResources
+	// TODO: Init counters and totals to MAX values
+	var totalBytes uint64
```

Comment:
You need to initialize the max allowed values for `totalBytes` and `batchZkCounters` I've created such function in the finalizer [getMaxRemainingResources](https://github.com/0xPolygonHermez/zkevm-node/blob/feature/newSequencer/sequencer/finalizer.go#L479) maybe will be better to move it to the sequencer as shared function we can reuse here as well.